src/vs/workbench/contrib/scm/browser/dirtydiffDecorator.ts

```diff
@@ -1101,25 +1085,26 @@ export async function getOriginalResource(quickDiffService: IQuickDiffService, u
 	return quickDiffs.length > 0 ? quickDiffs[0].originalResource : null;
 }
 
+type LabeledChange = { change: IChange; labels: string[]; uri: URI };
+
 export class DirtyDiffModel extends Disposable {
 
-	private _originalResource: URI | null = null;
-	private _originalModel: IResolvedTextEditorModel | null = null;
+	private _quickDiffs: QuickDiff[] | null = null;
```

Comment:
Should this now just be non-nullable, as well?

Reply:
Done.